apps/system/js/devtools/remote_debugger.js

```diff
@@ -35,35 +70,61 @@
       }
 
       var session = e.detail.session;
-      var text;
+      var title;
       if (!session.server.port) {
-        text = 'remoteDebuggerPromptUSB';
+        title = 'remoteDebuggerPromptUSB';
       } else {
-        text = {
+        title = {
           id: 'remoteDebuggerPromptTCP',
-          args: session.client
+          args: {
+            host: session.client.host,
+            port: session.client.port,
+            cert: session.client.cert.sha256.replace(/:/g, ' ')
```

Comment:
Please don't replace `:` with spaces. Instead, split the SHA-256 with something like:

    // Split the SHA256 cert into blocks of 24 chars.
    var sha = session.client.cert.sha256;
    var block = 24;
    var splitsha = [];
    for (var i = 0; i < sha.length; i += block) {
      splitsha.push(sha.slice(i, i + block));
    }

and then assign `title.args.cert` like so:

    cert: splitsha.join('\u200b')

This will add zero-width spaces at strategic places in the SHA-256 so that it can break and be displayed nicely.

Note: Please also display the SHA-256 in WebIDE using the same uppercase, `:`-separated format.